pkg/api/v20191231preview/openshiftcluster_convert.go

```diff
@@ -81,6 +81,17 @@ func (c *openShiftClusterConverter) ToExternal(oc *api.OpenShiftCluster) interfa
 		}
 	}
 
+	if oc.SystemData != nil {
```

Comment:
missing converter in the opposite direction, missing validation unit tests (fields should be considered immutable from the end user's perspective)

background: have tried to maintain an architectural separation so that the converters are always dumb, but there is validation that detects if a field has changed and it should not have done.